examples/app-mssql/main.py

```diff
@@ -0,0 +1,33 @@
+import pyodbc
+import sys
+import time
+
+# waiting for mssql db server to fully spawn
+time.sleep(3)
+
+print("Creating a mssql connection")
+conn = pyodbc.connect('DRIVER={ODBC Driver 17 for SQL Server};SERVER=mssql_server;DATABASE=tempdb;UID=SA;PWD=bad_P4ssword;')
+cur = conn.cursor()
+
+
+print("Create a new table: Fruits")
+cur.execute("CREATE TABLE Fruits (id INT, name NVARCHAR(50), quantity INT);")
```

Comment:
As for the general convention - let's use lowercase for table name.
```suggestion
cur.execute("CREATE TABLE fruits (id INT, name NVARCHAR(50), quantity INT);")
```